features/image-transformations/overlay-using-layers.md

```diff
@@ -0,0 +1,220 @@
+* [Transformation of text overlay](overlay-using-layers#transformation-of-text-overlay)
+* [Transformation of solid color overlay](overlay-using-layers#transformation-of-solid-color-overlay)
+
+Transformations inside a layer can be chained together to achieve the desired outcome.
+
+## Nesting of layers
+
+A layer can have a nested layer up to 3 levels. 
+
+For example, in the below URL, `i-inner.png` is rendered on the top-left corner of `i-outer.png` using the `lfo-top_left` parameter.
+
+```markup
+                          Parent layer
+┌──────────────────────────────────────────────────────────────┐
+l-image,i-outer.png,l-image,i-inner.png,lfo-top_left,l-end,l-end
+                    └──────────────────┬──────────────────┘
+                                       │
+                                  Nested layer
+```
+
+# Add images over image
+You can add an image over a base image using the following example.
+
+## Usage syntax
+
+```markup
+https://ik.imagekit.io/demo/base-image.jpg?tr=l-image,i-logo.png,l-end
+```
+
+You can also control the position of the image overlay using these [positional parameters](overlay-using-layers#position-of-layer).
+
+## Transformation of image overlay
+
+ImageKit supports many [image transformation parameters](../image-transformations/README.md). However, inside a layer, you can apply the following transformations to the image overlay before it is placed on the base image. You can [chain the transformations](../image-transformations/chained-transformations.md) one after the other to achieve the desired outcome.
+
+| Parameter                                                                                                   | Description                  |
+|-------------------------------------------------------------------------------------------------------------|------------------------------|
+| [w](../image-transformations/resize-crop-and-other-transformations#width-w)                                 | Width of the overlay image.  |
+| [h](../image-transformations/resize-crop-and-other-transformations#height-h)                                | Height of the overlay image. |
+| [ar](../image-transformations/resize-crop-and-other-transformations#aspect-ratio-ar)                        | Aspect ratio of the overlay image. |
+| [c](../image-transformations/resize-crop-and-other-transformations#crop-crop-modes-and-focus)               | Cropping method. Accepts `force`, `at_max`, and `at_least`. |
+| [cm](../image-transformations/resize-crop-and-other-transformations#crop-crop-modes-and-focus)              | Crop mode. Supports `extract` and `pad_resize`. |
+| [fo](../image-transformations/resize-crop-and-other-transformations#focus-fo)                               | Relative focus area used during cropping. Accepts `center`, `top`, `left`, `bottom`, `right`, `top_left`, `top_right`, `bottom_left`, and `bottom_right`. |
```

Comment:
Will `fo-face` and `fo-custom` not work in images?

Reply:
added